### DISCUSSION TITLE
MissionControl v0.4.1

### DISCUSSION THREAD

**Discussion by Anish5150** [ndeadly/MissionControl#217] [Q&A]
Thank you so much for your work. I am just wondering will MissionControl v0.4.1 on Firmware 11.0.1 on Atmosphere (v0.18.1) work or not ? Again thank you.

**Comment by ndeadly** [comment]
Yes, it works on those versions. In general, you can expect the latest release to support the most current firmware/atmosphere versions, unless there is a brand new update out that breaks something. You're more likely to encounter issues trying to run an outdated atmosphere version than anything.
